tests/saitests/py3/sai_qos_tests.py

```diff
@@ -1393,18 +1414,20 @@ def runTest(self):
             assert(recv_counters[pg] > recv_counters_base[pg]), \
                 'unexpectedly PFC counter not increase, {}'.format(test_stage)
             # recv port ingress drop
-            if platform_asic and platform_asic == "broadcom-dnx":
-                logging.info(
-                    "On J2C+ don't support port level drop counters - so ignoring this step for now")
-            else:
-                # recv port ingress drop
-                for cntr in ingress_counters:
-                    assert (recv_counters[cntr] > recv_counters_base[cntr]
-                            ), 'unexpectedly RX drop counter not increase, {}'.format(test_stage)
-                # xmit port no egress drop
-                for cntr in egress_counters:
-                    assert (xmit_counters[cntr] == xmit_counters_base[cntr]
-                            ), 'unexpectedly TX drop counter increase, {}'.format(test_stage)
+            # For dnx few extra ipv6 NS/RA pkt received from VM, adding to counter value
```

Comment:
@ansrajpu-git  Original code is stating these counters not supported for DNX. Is this not the case ?

Reply:
> @ansrajpu-git Original code is stating these counters not supported for DNX. Is this not the case ?

vineet, these drop counters were not supported initially and they are supported now. So enabling the check